scripts/pre-upgrade/CASMREL-776-CSM12-NCN-boot-order-backport/install-hotfix.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+# Copyright 2021 Hewlett Packard Enterprise Development LP
+
+set -o errexit
+set -o pipefail
+
+WORKING_DIR=$(dirname $0)
+pitinit_ver='1.2.11-1'
+ilorest_ver='3.2.3-1'
+# Installs RPMs, or fetches them if possible.
+function install_rpms {
+    zypper --no-gpg-checks \
+        # FIXME: Remove this commented line and figure out the repository name.
```

Comment:
@zcrisler or @denniswalker what should this URL be? I am not familiar, but how can I deduce this?

Reply:
You can reference the latest CSM SLE 15-SP2 repository from the most recent install via https://packages.local/repository/csm-sle-15sp2